language-design.md

```diff
@@ -17,7 +17,81 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+***Finding the Balance in Creation: Cathedral versus Bazaar***
+
+> The key point is that in the bazaar style of building a program or designing a
+> language or what you will, the plan can change in real time to meet the needs
+> of those who work on it and use it. This tends to make users stay with it as
+> time goes by; they will take joy in working hard and helping out if they know
+> that their wants and needs have some weight and their hard work can change the
+> plan for the better [Steele, 1998]
+
+While flexibility as emphasized in this quote and throughout Steele's paper
+are foundational to good software development and general-purpose programming
+language development, we feel that a stricter approach when building out a DSL
+is essential in avoiding feature-bloat. Anthony pointed out an analagous duality
+of development styles prompted by this quote: Cathedral-Style is to Bazaar-Style
+as the [Waterfall Model](https://en.wikipedia.org/wiki/Waterfall_model) is to
+the [Agile Model](https://en.wikipedia.org/wiki/Agile_software_development).
+The former one, through a deep design and requirement definition process,
+produces a sort of master plan while the other is a more itereative,
+adaptive approach even once building (or coding) has started. While the Agile
+model can be extremley useful in more general software projects, for a DSL a
+so-called "master plan" can be extemely useful.
```

Comment:
All projects can benefit from a master plan, at least one that provides a general roadmap and goals. The Agile model doesn't necessarily preclude having a direction or a goal; it just means working iteratively towards that goal and adapting to meet changing requirements.